pike/test/copychunk.py

```diff
@@ -150,6 +171,90 @@ def _prepare_source_file(self):
         src_filename = "src_copy_chunk_rk.txt"
         self._create_and_write(src_filename, SIMPLE_CONTENT)
 
+    def _read_big_file(self, file_handle, total_len, add_offset=0):
```

Comment:
This is good stuff. I would consider this type of function for inclusion in the base pike.model.Channel class